content/metadata.yaml

```
@@ -65,9 +65,12 @@ authors:
   -
     github: rasbt
     name: Sebastian Raschka
+    twitter: rasbt
     initials: SR
     orcid: 0000-0001-6989-4493
     email: sraschka@wisc.edu
+    funders: Wisconsin Alumni Foundation (AAD5912)
```

```suggestion
    funders:
      - Wisconsin Alumni Foundation (AAD5912)
```
I believe the funders info has to be a list even if there is only one.

---

Thanks!